website/docs/cloud-docs/projects/index.mdx

```diff
@@ -0,0 +1,48 @@
+---
+page_title: Projects - Terraform Cloud
```

Comment:
I love this page separation/reorder; thank you for giving projects their own section! ❤️